src/components/UserAvatar.vue

```diff
@@ -24,14 +24,13 @@ watchEffect(() => {
     <UiPopover :options="{ offset: [0, 12], placement: 'bottom-start' }">
       <template v-slot:item>
         <a class="flex flex-nowrap">
-          <UiAvatar
+          <BaseAvatar
             v-if="!onlyUsername"
             :address="address"
             size="18"
             class="mr-2"
           />
-          <span class="w-[50px] truncate sm:hidden">{{ fullUserName }}</span>
-          <span class="truncate hidden sm:block">{{ username }}</span>
+          <span class="truncate w-[50px] sm:w-full">{{ username }}</span>
```

Comment:
Any reason you removed `fullUserName`? That was to avoid having addresses truncated like this on mobile. `0x123...45...`. I didn't like the implementation a lot tbh but... it did something at least. :D

Reply:
Fixed it like this now on `SpaceProposal` and `SpaceProposals` page

<img width="363" alt="image" src="https://user-images.githubusercontent.com/51686767/157216390-9fe11f1c-abac-46b6-b712-c8a792f1ae3e.png">

<img width="351" alt="image" src="https://user-images.githubusercontent.com/51686767/157216674-5b64fa9f-d497-4fd9-86e8-88a05ecc9cbc.png">